core-libs/setup/tsconfig.spec.json

```diff
@@ -637,22 +637,50 @@
       "@spartacus/opf/checkout/components": [
         "../../integration-libs/opf/checkout/components/public_api"
       ],
-      "@spartacus/opf/checkout/core": [
-        "../../integration-libs/opf/checkout/core/public_api"
-      ],
       "@spartacus/opf/checkout": [
         "../../integration-libs/opf/checkout/public_api"
       ],
-      "@spartacus/opf/checkout/occ": [
-        "../../integration-libs/opf/checkout/occ/public_api"
-      ],
       "@spartacus/opf/checkout/root": [
         "../../integration-libs/opf/checkout/root/public_api"
       ],
+      "@spartacus/opf/cta/core/components": [
```

Comment:
[MEDIUM]
Usually `/components` is a sibling to `/core`.
Here it's nested under `/core`. Was it a deliberate decision or a "typo"?

Reply:
It was a "typo". I moved "components" folder along the "core" and moved `OpfCtaComponentsModule` import into opf-cta.module.ts